packages/server/utils/analytics/analytics.ts

```diff
@@ -477,8 +525,29 @@ class Analytics {
     })
   }
 
-  accountRemoved = (userId: string, reason: string, parabolPayload: object) => {
-    this.track(userId, 'Account Removed', {reason, parabolPayload})
+  accountRemoved = async (userIdToDelete: string, email: string, validReason: string) => {
+    const executeGraphQL = require('../../executeGraphQL').default
+    const parabolPayload = await executeGraphQL({
```

Comment:
-1 is there a way to track `accountRemoved` without importing the graphql executor here? 

It looks like it's only being used in `hardDeleteUser`. Could that mutation query all the data we need and then we pass it to `accountRemoved` as args?